File: ADSO/Lenguajes/JavaScript/Act_29-07-24.js
class Dog{
    constructor(raza, edad, peso){
        this.raza = raza;
        this.edad = edad;
        this.peso = peso;
    }
}

class Moto {
    constructor(marca, color, modelo) {
      this.marca = marca;
      this.color = color;
      this.modelo = modelo;
    }
}
  
class Peli {
    constructor(genero, edadMaxPermitida, fechaDeEstreno) {
      this.genero = genero;
      this.edadMaxPermitida = edadMaxPermitida;
      this.fechaDeEstreno = fechaDeEstreno;
    }
}


const a = new Dog("Doberman", 7, 35);

const b = new Moto("Suzuki", "Negro", "DR 150 FI");

const c = new Peli("Acción", 18, 25072024);

console.log(a)
console.log(b)
console.log(c)

File: ADSO/Lenguajes/Dart/Act_29-07-24.dart
class Dog{
  String? raza;
  int? edad;
  int? peso;
}

class Moto{
  String? marca;
  String? color;
  String? modelo;
}

class Peli {
  String? genero;
  int? edadMaxPermitida;
  int? fechaDeEstreno;
}


void main(){
  Dog a = new Dog();

  a.raza = "Doberman";
  a.edad = 7;
  a.peso = 35;

  Moto b = new Moto();

  b.marca = "Suzuki";
  b.color = "Negro";
  b.modelo = "DR 150 FI";

  Peli c = new Peli();

  c.genero = "Acción";
  c.edadMaxPermitida = 18;
  c.fechaDeEstreno = 25072024;

  
}